packages/ui/src/2_molecules/NavMenuItem/NavMenuItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, ReactNode, useMemo, useRef } from 'react';
+
+import classNames from 'classnames';
+
+import { Badge } from '../../1_atoms/Badge/Badge';
+import styles from './NavMenuItem.module.css';
+
+const MAX_NOTIFICATIONS_COUNT = 99;
```

Comment:
```suggestion
const MAX_COUNT_DEFAULT = 99;
```
I don't like this because you can basically never override it, unless you change it in the UI package and redeploy it. I'd add a property `maxNotificationsCount` that will be optional and its default value would be MAX_COUNT_DEFAULT.